docs/learn/calls.md

```diff
@@ -70,7 +78,7 @@ Mirascope's call decorator supports multiple LLM providers, allowing you to easi
 
 To use a specific provider, simply use the `call` decorator from the corresponding provider's module. Here's an example of how to use multiple different providers with the same function:
```

Comment:
nit: we should update the last bit "the same function" to "the same prompt template"